src/puppetlabs/ezbake/core.clj

```diff
@@ -427,7 +442,8 @@ Dependency tree:
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
                                                           "-Xmx192m")
-         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
+                                    ; Convert to string so ruby doesn't barf on the hyphens
+         :bootstrap-source          (name (get-local-ezbake-var lein-project :bootstrap-source :bootstrap-cfg))}))))
```

Comment:
I did that initially, but testing it would require calling this `generate-ezbake-config-file`, and creating all the parameters that need to be passed in. I guess it wouldn't be much worse. The other reason is that putting it in `action` causes the stage command to fail immediately, before doing all the file copying. Saves a good 10 seconds, but yeah, probably better to move it back here